### ISSUE TITLE
cumulus-bootstrap script relies on deprecated command

### ISSUE THREAD

**Issue by acarpi** [Apstra/aeon-ztps#137]
Trying to use Aeon ZTP to upgrade from Cumulus 3.1.1 to Cumulus 3.1.2, the process failed in the bootstrap script.

Apparently, this command is not present in Cumulus 3.1.1

```
/usr/cumulus/bin/cl-img-install -sf http://172.20.197.10:8080/vendor_images/cumulus/cumulus-linux-3.1.1-bcm-amd64.bin
-bash: /usr/cumulus/bin/cl-img-install: No such file or directory
```
I opened a support case with Cumulus to understand what could be a suitable replacement.

cc @jeremyschulman

**Comment by acarpi** [comment]
Got answer from cumulus, there is no way we can select the destination image from the provisioning script, since the concept of slot is deprecated in 3.x

The new (and backward incompatible) way to install an image is:
1. onie-select -rf
2. Reboot, the box will go in ONIE rescue mode
3. SSH into the box with the ONIE default credentials, root@<cumulus_ip>, no password
4. run onie-nos-install http://path/to/onie-installer
5. Box will reboot again, it will install the image
6. After ~10min installation will be completed, new reboot

**Comment by bobbywatson3** [comment]
@acarpi that process looks really gross to try to automate. Is that the only way?

**Comment by bobbywatson3** [comment]
@acarpi I'm working on this bug now. I think I may have found an easier process. I'll let you know how it goes.

**Comment by bobbywatson3** [comment]
I have fixed the issue and tested in slicer. I've also created a qcow2 image that @acarpi is testing.

**Comment by bobbywatson3** [comment]
Resolved in #144
